_includes/head-idr.html

```diff
@@ -44,7 +44,7 @@
                             <li><a href="{{ "/api.html" | prepend: site.baseurl }}">API Access</a></li>
                             <li><a href="{{ "/download.html" | prepend: site.baseurl }}">Data download</a></li>
                             <li><a href="{{ "/itr.html" | prepend: site.baseurl }}">Image Tools Resource (ITR)</a></li>
-                            <li><a href="/jupyter">Virtual Analysis Environment (VAE)</a></li>
+                            <li><a href="https://mybinder.org/v2/gh/IDR/idr-notebooks/master">Virtual Analysis Environment (VAE)</a></li>
```

Comment:
While updating the menu makes sense, we are still contemplating to redirecting these resources to the EBI Binder resources when these achieve production status.

Another solution would be to continue using an intermediate URL and adjust the redirects under the hood as we migrate?

Reply:
We could.
If EBI writes some documentation about its deployment, we may, at stage, add a new entry,e.g. start the VAE and keep the <li><a href="/jupyter">Virtual Analysis Environment (VAE)</a></li> to deployment of EBI doc.